### PR TITLE
fix(widget-ui): widget UI improvements

### PR DESCRIPTION
**Pull request by Cali93**
Closes https://github.com/WalletConnect/web3inbox-widget/issues/9

### CODE REVIEW

File: packages/ui/src/components/w3i-widget/index.ts [hunk 1]
```diff
@@ -142,9 +101,6 @@ export class W3iWidget extends LitElement {
 
     return html`
       <div ${ref(this.iframeRef)} class="w3i-widget">
-        <button class="w3i-close-button" @click=${this.onCloseClick}>
-          <div class="w3i-close-button-foreground"></div>
-        </button>
```
Comment (devceline) [inline packages/ui/src/components/w3i-widget/index.ts:147]: I know this was supposed to be replaced by a listener that checks for clicks outside the widget, but I don't see that here?

Comment (Cali93) [reply packages/ui/src/components/w3i-widget/index.ts:147]: Never heard of that requirement. Is there an issue for it ?
Anyway, that can be handled by the dapp that implements the widget thanks to your new hooks.
Also we can close the widget by clicking the bell and that X button isn't on any design so better remove it.

**Review by devceline — APPROVED** [review]
LGTM

File: packages/ui/src/components/w3i-widget/index.ts (hunk 1 above)
Comment (devceline) [reply packages/ui/src/components/w3i-widget/index.ts:147]: Ya this is solved easily with the hooks, lgtm

**Pull request merged**